pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -554,7 +578,7 @@ func getSmartRouterTemplate(cr *api.KieApp) api.SmartRouterTemplate {
 }
 
 // GetImage ...
-func GetImage(imageURL string) (image, imageTag, imageContext string) {
+func GetImage(imageURL string) (image, imageTag, imageContext string, error error) {
```

Comment:
instead of error, should we use a `bool` here? another option i've considered is to modify the function to properly return image/imagetag even if the image is using a digest. so the tag returned would be the proper `sha256:xxxxxxxxcb244ea5f127f28d36fa88069f864f54xxxxx` if an `@` is detected

Reply:
also, given that labels will no longer be leveraging this `GetImage` function... any changes to this function should probably be done separately. please move changes specific to this function to a new PR.

Reply:
I'll change in a different PR